src/test/java/org/dice_research/opal/civet/metrics/DateFormatMetricTest.java

```diff
@@ -0,0 +1,84 @@
+package org.dice_research.opal.civet.metrics;
+
+import org.apache.jena.ext.com.google.common.annotations.VisibleForTesting;
+import org.apache.jena.rdf.model.Model;
+import org.dice_research.opal.civet.TestData;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests {@link Dateformat}.
+ * 
+ * @author Aamir Mohammed
+ */
+public class DateFormatMetricTest {
+
+	@VisibleForTesting
+	TestData testdata;
+	@VisibleForTesting
+	final String MODEL = "TestDateFormatMetric.ttl";
+	@VisibleForTesting
+	Model model;
+
+	@Before
+	public void setUp() throws Exception {
+		testdata = new TestData();
+		model = testdata.getModel(MODEL);
+	}
+
+	@Test
+	public void correctFormat() throws Exception {
+
+		DateFormatMetric metric = new DateFormatMetric();
+		final String datasetUri = "http://projekt-opal.de/dataset/_mcloudde_standortederdauerzhlstellenradverkehrindsseldorf";
+		Integer stars = metric.compute(model, datasetUri);
+		Assert.assertEquals("It contains all correct format", 5, stars.intValue());
```

Comment:
add a short comment: which number of correct and incorrect formats in dataset / distributions?

Reply:
Okay